proco/connection_statistics/utils.py

```diff
@@ -97,15 +94,16 @@ def _get_start_and_end_date_from_calendar_week(year, calendar_week):
     return monday, monday + timedelta(days=6.9)
 
 
-def aggregate_country_daily_status_to_country_weekly_status(date=None):
+def aggregate_country_daily_status_to_country_weekly_status(country_id, date=None):
     date = date or timezone.now().date()
     week_ago = date - timedelta(days=7)
-    countries = Country.objects.filter(
+    country = Country.objects.filter(
+        id=country_id,
         id__in=CountryDailyStatus.objects.filter(
             date__gte=week_ago,
         ).order_by('country__name').values_list('country', flat=True).order_by('country_id').distinct('country_id'),
-    )
-    for country in countries:
+    ).last()
```

Comment:
let's use .get() here. it's good to know when we pass incorrect country id because usually we don't remove them